default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib, fetchFromGitHub, buildDunePackage, ansiterminal, sedlex_2, menhir
+, unionfind, bindlib, cmdliner, re, zarith, zarith_stubs_js, ocamlgraph
+, calendar, visitors, benchmark, js_of_ocaml, js_of_ocaml-ppx, camomile, cppo }:
+
+buildDunePackage rec {
+  pname = "catala";
+  version = "0.3.0";
+
+  minimumOCamlVersion = "4.08";
+
+  src = ./.;
+
+  useDune2 = true;
+
+  propagatedBuildInputs = [
+    ansiterminal
+    sedlex_2
+    menhir
+    cmdliner
+    re
```

Comment:
Same remark, I guess there's no tool that can generate that from `dune-project` ?

Reply:
I should be able to make a small python script that parses dune-project and put the constraints as asserts in the default.nix file.